generation/localizedstringkit/__init__.py

```diff
@@ -16,35 +16,44 @@
 from localizedstringkit.files import localizable_files
 
 
-SOURCE_CODE_FILE_NAME: str = "source_strings.m"
-BUNDLE_NAME: str = "LocalizedStringKit.bundle"
```

Comment:
Moving forward, the `SOURCE_CODE_FILE_NAME` will be the <bundle_name>.m. Also, the `BUNDLE_NAME` will no longer be hardcoded.